kuma/users/tests/test_views.py

```diff
@@ -1602,3 +1602,16 @@ def test_invalid_uid_fails(wiki_user, client):
     assert response.status_code == 200
     assert_no_cache_header(response)
     assert b'This link is no longer valid.' in response.content
+
+
+def test_signin_landing(db, client, settings):
+    settings.MULTI_AUTH_ENABLED = True
+    response = client.get(reverse('socialaccount_signin'))
+    assert response.status_code == 200
+    doc = pq(response.content)
+    assert 'Create your MDN Web Docs Account' in doc.find('h1').text()
```

Comment:
How is this test different from the one in `test_templates.py`?

Actually, I'm not entirely sure what `test_TEMPLATES.py` mean because what really matters is what users SEE and that API is the view. Not the template. 
I'd vote you *move* that nice `test_signin_landing` into this file here.

Reply:
¯\_(ツ)_/¯ - I was doing some searching around the tests and stumbled on one of these `test_template` files. My thinking then was, perhaps the `view` tests, in this case, should just test that the appropriate response is received based on the `MULTI_AUTH_ENABLED` setting.

The actual specifics of what should be present in the page is then tested in the `template` tests. Perhaps these are just a holdover or are used for specific instances. Not sure. I will move the contents of that test here.